sources/visual/SoundRect.cpp

```diff
@@ -571,3 +574,8 @@ void SoundRect::describe(std::ostream &out)
     out << "- X : " << getX() << "\n";
     out << "- Y : " << getY() << "\n";
 }
+
+string SoundRect::getName()
```

Comment:
Retour de chaîne sans copie

```suggestion
const string &SoundRect::getName() const
```